dali/ft/daftformat.cpp

```diff
@@ -287,9 +287,18 @@ void CInputBasePartitioner::findSplitPoint(offset_t splitOffset, PartitionCursor
         ensureBuffered(headerSize);
         assertex((headerSize ==0) || (numInBuffer != bufferOffset));
 
-        bool processFullBuffer =  (nextInputOffset + blockSize) < splitOffset;
+        unsigned size = 0;
+        if ((nextInputOffset + thisSize) <= splitOffset)
+            // If there are many small files to spray and the
+            // current file size (thisSize) is fit into this part then
+            // store it without process.
+            size = thisSize;
```

Comment:
Sounds good, but was this something that changed in a 6.x build that needs to go into a point build?
If not, should move this to target next version.

Reply:
I can agree and I had same thought. So I  will remove this part and raise a separate JIRA to optimize partition calculation for large amount but small size files.